lib/openhab/dsl/items/ensure.rb

```diff
@@ -81,11 +81,15 @@ def <<(command)
         end
 
         # activate +ensure_states+ before forwarding to the wrapped object
-        def method_missing(method, *args, &block)
+        def method_missing(method, *args, **kwargs, &block)
```

Comment:
the proper way to fix this is with ruby2_keywords. normally you'd run into issues with ruby 2.7, but since jruby is skipping ruby 2.7, this _might_ be okay, but still concerns me

Reply:
I've pushed a version that uses ruby2_keywords. It passed the jruby94 test too